api/modules/currency/views.py

```diff
@@ -33,3 +35,51 @@ def get_currency_exchange_rate(request, source_currency_code, target_currency_co
         return Response(exception_message, status=status.HTTP_503_SERVICE_UNAVAILABLE)
 
     return Response(response.to_json())
+
+
+@api_view(['GET'])
+def get_all_currency_exchange_rate(request, start_date, end_date, source_currency_code, target_currency_code):
+    """
+    Return currency exchange rates list between 2 dates
+    :param request:
+    :param start date:
+    :param end date:
+    :param source_currency_code:
+    :param target_currency_code:
+    :return 400 response if dates are incorrect
+    :return: 503 if Free Currency Converter api fails
+    :return: 200 successful
+    """
+    currency_list = []
+    y_start = start_date.split('-')[0]
+    m_start = start_date.split('-')[1]
+    d_start = start_date.split('-')[2]
+    start = datetime.datetime.strptime(y_start+m_start+d_start, '%d%m%Y').date()
```

Comment:
For 14th July 2018 (`start_date = "14-07-2018"`), you can use:
`datetime.datetime.strptime(start_date, "%d-%m-%Y")`
Not completely sure about the format, but something on these terms, you don't need to split it yourself `strptime` can do it for you
Reference: https://stackoverflow.com/a/466376/5602759

Reply:
thats not working